### ISSUE TITLE
Plans to re-integrate gesture support?

### ISSUE THREAD

**Issue by frankwangyufei** [Errrneist/Hackintosh-Thinkpad-X1-Extreme#16]
First things first, great work on this!

Right now I kind of can get 3 finger gesture to work by remapping stuff to ctrl+cmd+arrow_key.

Still, is there plan to support gesture again like release 10.14.3.1.2?

**Comment by wu-hongjun** [comment]
Currently, there are two ways to approach this problem. One is via ApplePS2SmartTouchpad.kext, and one is via VoodooPS2Controller.kext. We picked the second approach since it is the most stable approach, the first approach usually gives weird errors and may cause a kernel panic. Due to my limited knowledge, I don't know how to modify VoodooPS2Controller.kext, so we will have to go with what is the default. 

If you want to trade TrackPoint off for multi-gesture (Which I actually don't use multi-gesture very often, that is why I rolled it back), then download the first 10.12.3 release and replace the VoodooPS2Controller.kext with that one. It should work.
